src/styles.scss

```diff
@@ -2,3 +2,5 @@
 @import "_component/_layout/roboto";
 @import "./_component//about-me/about-me.scss";
 @import "./_asset/font/style.scss";
+@import "./_component/_connect/connect.scss";
```

Comment:
Be careful with these imports and try to use the generic ones at the top and the most specific ones at the bottom; like the one in which you import the font first and then the one responsible the about-me section